src/Libraries/Nop.Core/Domain/Catalog/ProductEditorSettings.cs

```diff
@@ -232,6 +232,11 @@ public class ProductEditorSettings : ISettings
         /// </summary>
         public bool AllowAddingOnlyExistingAttributeCombinations { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to display attribute combination images only
+        /// </summary>
+        public bool DisplayAttributeCombinationImagesOnly { get; set; }
```

Comment:
the setting is used only to control visibility on the product editing page, which is not entirely correct, since when it is disabled, all products for which the analogous property was set to true will retain their behavior, which can confuse users